src/3box.js

```diff
@@ -506,11 +510,7 @@ class Box {
     }
 
     // Send consentSignature to 3box-address-server to link profile with ethereum address
-    try {
-      await utils.fetchJson(this._serverUrl + '/link', linkData)
-    } catch (err) {
-      console.error(err)
-    }
+    utils.fetchJson(this._serverUrl + '/link', linkData).catch(console.log)
```

Comment:
Why console.log and not console.error? Or maybe warn?

Reply:
yeah just changed that

Reply:
haha, you fixed it before I finished the review :P

Reply:
we are too quick now!

tests still not happy about it yet